### PR TITLE
travis improvements

### PR DESCRIPTION
**Pull request by oetiker**
* make travis run stage scripts in parallel
* add chrome test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -44,14 +40,17 @@ jobs:
...
     - stage: Lint-and-Coverage
       script: ../.travis/lint
+    - # parallel
       script: ../.travis/coverage
-    - stage: Firefox-Unit-Tests-Local
+    - stage: Local-Browser-Tests
       script: ../.travis/test Firefox
-    - stage: Chrome-Unit-Tests
+    - # parallel
+      script: ../.travis/test Chrome_travis_ci
```
Comment (level420) [inline .travis.yml]: The reason why (it was me) the tests were run in stages one after the other was that the tests sometime fail with timeouts caused by slow running containers at travis or saucelab. If the first test in firefox succeeded and the chrome test failed afterwards it was sufficent to restrat the chrome test via the travis web interface until it succeeded. If you are running firefox and chrome in parallel the probability that one of the test fails and you have to rerun the stage with both tests again is higher. The drawback is that the test will take a longer time if the tests run sequentially in stages.

Comment (level420) [reply .travis.yml]: I also suspect that the travis test have cpu quota which potentially lead to container run interruptions or lower cpu priority if you run tests in parallel, at least if you are a non paying customer.

**Review by level420 — DISMISSED** [review]

**Review by level420 — CHANGES_REQUESTED** [review]
See comments above

File: .travis.yml (hunk 1 above)
Comment (oetiker) [reply .travis.yml]: you can restart the individual jobs within a stage

**Comment by oetiker** [comment]
these tests also 'enable' the lint tests ... in the old setup they were in .travis but since you can not have 2 script keys in the same hash they were inactive!

**Review by level420 — APPROVED** [review]

**Comment by oetiker** [comment]
merging this to get the CI back into play

**Pull request merged**